cmd/config-manager/main.go

```diff
@@ -43,7 +44,26 @@ var (
 	log = logrus.StandardLogger()
 )
 
+type nriConfig struct {
+	registrationTimeout string
+	requestTimeout      string
+}
+
 func main() {
+	var cfg nriConfig
+
+	flag.StringVar(&cfg.requestTimeout,
+		"nri-plugin-request-timeout", "", "configure NRI plugin request timeout")
+	flag.StringVar(&cfg.registrationTimeout,
+		"nri-plugin-registration-timeout", "", "configure NRI plugin registration timeout")
```

Comment:
this is a nit, feel free to just ignore it. Perhaps If it was me, I would have dropped the `nri` prefix to make those args shorter

Reply:
It is a valid point and I was considering that, too. Why I eventually chose these command line options is that these are literally the same as the corresponding command line options for CRI-O (in containerd you don't have fine-grain configuration control using the command line).